src/Provider/ImageProvider.php

```diff
@@ -71,6 +71,8 @@ public function getHelperProperties(MediaInterface $media, $format, $options = [
             throw new \LogicException("The 'srcset' and 'picture' options must not be used simultaneously.");
         }
 
+        $attrPrefix = isset($options['lazy']) && true === $options['lazy'] ? 'data-' : '';
```

Comment:
Shouldn't there be some kind of validation somewhere? IMO there should be an exception if `$options['lazy']` is not a boolean.

Reply:
I was thinking of allowing an attribute map as well, something like 

```
{
  lazy: {
    src: data-src,
    srcset: data-srcset
  }
}
```

Is that something we should add in here?

Reply:
> Perhaps in a different key though because it’s more generic

But then you would have different keys, which is not ideal I think. Do you think people really will want to customize this (I have no idea)?

Reply:
Probably not, as I wrote earlier all implementations I came across use `data-` attributes. It would only allow to use implementations that don’t follow this or just rename attributes for totally different use cases

Reply:
Let's go for "totally different use cases", and change `lazy` into something else.